commit d8e33cd84ae01ca281d497e390101c79e03f70a1
Author: Anatol Pomozov <anatol.pomozov@gmail.com>
Date:   Wed Dec 25 15:35:42 2019 -0800

    Fix ruby 2.7 keyword parameter warning
    
    warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call

diff --git a/lib/hoe/debug.rb b/lib/hoe/debug.rb
index 8bfdcf1..67be47b 100644
--- a/lib/hoe/debug.rb
+++ b/lib/hoe/debug.rb
@@ -91,7 +91,7 @@ module Hoe::Debug
       begin
         sh "#{DIFF} -du Manifest.txt #{f}", verbose
       ensure
-        rm f, verbose
+        rm f, **verbose
       end
     end
   end
